docs/_docs/windows.md

```diff
@@ -171,3 +171,13 @@ In the future the installation process of the github-pages should be as simple a
 [Bundler]: http://bundler.io/ "Ruby Dependencie Manager"
 [nokogiriReleases]: https://github.com/sparklemotion/nokogiri/releases "Nokogiri Releases"
 [nokogiriFails]: https://github.com/sparklemotion/nokogiri/issues/1456#issuecomment-206481794 "Nokogiri fails to install on Ruby 2.3 for Windows"
+
+## Installation via RubyInstaller
+
+RubyInstaller is a self-contained Windows-based installer that includes the Ruby language, an execution environment, important documentation, and more.
+
+1. Install a package manager for Windows called [RubyInstaller](https://rubyinstaller.org/).
+2. Install Jekyll via RubyInstaller: `gem install bundler jekyll`
```

Comment:
I haven't tried the latest version from RubyInstaller2, but does the Installer itself have an environment to accept shell commands?
I feel L#180 and L#181 should rather be:
```
2. Install Jekyll and Bundler via a command prompt window: `gem install jekyll bundler`
3. Check if the installation is accessible: `jekyll -v`
```